Updated for brevity and link to precompiled samples

AbhishekTripathi opened the pull request:
I have updated the examples to have the header as `C# script` to avoid any confusion. Also, added is an attention to the first example to highlight the subtle difference between the C# script and the precompiled function with respect to the usage of attributes.

AbhishekTripathi committed:
Updated for brevity and link to precompiled samples

I have updated the examples to have the header as `C# script` to avoid any confusion. Also, added is an attention to the first example to highlight the subtle difference between the C# script and the precompiled function with respect to the usage of attributes.

cgillum commented on articles/azure-functions/durable-functions-overview.md:
````diff
@@ -57,6 +57,8 @@ public static async Task<object> Run(DurableOrchestrationContext ctx)
 }
 ```
+> [!NOTE]
+> Above example is for C# script as indicated. There are subtle differences while writing a precompiled durable function in C# vs the C# script sample shown above. A C# precompiled function would require durable parameters to be decorated with respective attributes. An example is ```[OrchestrationClient]``` attribute for ```DurableOrchestrationClient``` parameter. If the parameters are not proper ...
````
- Instead of "Above example", use "The previous example" ("above" doesn't make sense for folks who use screen readers)
- I would change the link to (https://github.com/Azure/azure-functions-durable-extension/blob/master/samples, rather than pointing to HttpStart.cs
- The example here doesn't show `DurableOrchestrationClient`, so I would instead use `[OrchestrationTrigger]` and `DurableOrchestrationContext` as your examples.
- Optional, consider removing the sentence starting with "If the parameters are not properly decorated...". I'm just worried it might be too much text. The fact that you mentioned the difference in requirements for precompiled C# seems sufficient to me - but again...optional.

cgillum reviewed (commented):
Thanks for this PR! It's definitely appreciated. I have a few minor suggestions for your callout before approving.

AbhishekTripathi committed:
Updated language

The verbiage is updated keeping screen reader users as mentioned in review comments.

AbhishekTripathi committed:
Updated attribute to match the code sample

Updated attribute example to match the previous code sample.

cgillum commented on articles/azure-functions/durable-functions-overview.md:
````diff
@@ -57,6 +57,8 @@ public static async Task<object> Run(DurableOrchestrationContext ctx)
 }
 ```
+> [!NOTE]
+> There are subtle differences while writing a precompiled durable function in C# vs the C# script sample shown before. A C# precompiled function would require durable parameters to be decorated with respective attributes. An example is `[OrchestrationTrigger]` attribute for `DurableOrchestrationContext` parameter. If the parameters are not properly decorated, the runtime would not be able to inj ...
````
There is an extra `(` in your markdown where you link to samples.

AbhishekTripathi committed:
Too many brackets :((

cgillum reviewed (approved):
Looks good!

#sign-off
